packages/strapi-plugin-users-permissions/services/UsersPermissions.js

```diff
@@ -0,0 +1,250 @@
+'use strict';
+
+const fs = require('fs')
+const path = require('path');
+const stringify = JSON.stringify;
+const _ = require('lodash');
+// const Service = strapi.plugins['users-permissions'].services;
+/**
+ * UsersPermissions.js service
+ *
+ * @description: A set of functions similar to controller's actions to avoid code duplication.
+ */
+
+module.exports = {
+  createRole: (role) => {
+    const Service = strapi.plugins['users-permissions'].services.userspermissions;
+    const appRoles = require(Service.getRoleConfigPath());
```

Comment:
The roles are already loaded in the global variable. `strapi.plugins['user-permissions'].config...`. However to load them in the `roles` key, you need to encapsulate the file content in a `roles` key.